scripts/installation/install-deps-alpine.sh

```diff
@@ -0,0 +1,184 @@
+#!/bin/sh
+
+# Comprehensive dependency installation script for Tracee (Alpine Linux)
+# Replaces .github/actions/build-dependencies for containerized workflows
+
+set -e
+
+# Source lib.sh for consistent logging and utilities  
+SCRIPT_DIR=$(cd "$(dirname "$0")" && pwd)
+# shellcheck disable=SC1091
+. "$SCRIPT_DIR/../lib.sh"
+
+info "Starting Tracee dependency installation on Alpine Linux"
+
+# Configuration
+GOLANG_VERSION="1.24.0"
+STATICCHECK_VERSION="2025.1"
+REVIVE_VERSION="v1.7.0"
+GOIMPORTS_REVISER_VERSION="v3.8.2"
+ERRCHECK_VERSION="v1.9.0"
+
+install_base_packages() {
+    info "Installing base packages"
+    require_cmds apk
+    
+    apk update
+    apk add --no-cache \
+        bash \
+        build-base \
+        sudo \
+        coreutils \
+        findutils \
+        git \
+        curl \
+        rsync \
+        make \
+        gcc \
+        musl-dev \
+        linux-headers \
+        elfutils-dev \
+        libelf-static \
+        zlib-static \
+        zstd-static \
+        libc6-compat \
+        tar \
+        ca-certificates
+
+    # Create symlinks for compatibility with tests expecting binaries in /usr/bin
+    # BusyBox applets - link directly to busybox so applet name is detected correctly
+    ln -sf /bin/busybox /usr/bin/uname
+    ln -sf /bin/busybox /usr/bin/date
+
+    info "Base packages installed successfully"
+}
+
+install_golang() {
```

Comment:
for the future: may be extracted to a `install-go.sh` as it's repeated in the ubuntu as well. Even if not equal, it could follow as the `install-clang.sh` which already handles alpine and ubuntu.

Reply:
good idea!